src/btree/bt_vrfy.c

```diff
@@ -1049,18 +1051,11 @@ __verify_page_content(
                   __wt_timestamp_to_string(unpack.newest_stop_ts, ts_string[0]),
                   __wt_timestamp_to_string(unpack.newest_stop_durable_ts, ts_string[1]));
 
-            WT_RET(__verify_ts_addr_cmp(session, ref, cell_num - 1, "start durable",
-              unpack.newest_start_durable_ts, "start durable", addr_unpack->newest_start_durable_ts,
-              false, vs));
```

Comment:
Why aren't these checks valid any more? I thought they would be.

Reply:
During the verify flow, sometimes when the internal page is not reconciled, it is possible that the internal page cell is not updated with the latest timestamp information from the child. It is same with the other fields also, but the other fields comparison always tends to true even with the internal page cell has default values, but that's not true with durable timestamps. 

I can either change the comparison to be done only when the {{addr_unpack}} durable timestamps when they do not default values. That can fix the problem. Is that fine?

Reply:
I see you made the change - but yes, that would be fine.